bin/cli.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
```

Comment:
This isn't as well developed as the CLI set-up for other repos but it does the job for now.